### ISSUE TITLE
failing installing  simple-git-hooks npm package

### ISSUE THREAD

**Issue by erossignon** [pnpm/pnpm#3287]
### pnpm version:
5.18.8
### Code to reproduce the issue:

$ mkdir demo
$ cd demo 
$ npm init -y
$ pnpm install simple-git-hooks

### Expected behavior:

Expecting the same behavior as with npm: successful install

### Actual behavior:

```console
PS C:\temp\demo> pnpm install simple-git-hooks --save-dev
Packages: +1
+
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: C:\Users\me\.pnpm-store\v3
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 1, reused 1, downloaded 0, added 1, done
node_modules/.pnpm/simple-git-hooks@2.2.0/node_modules/simple-git-hooks: Running postinstall script, failed in 62ms
.../node_modules/simple-git-hooks postinstall$ node ./postinstall.js
│ internal/fs/utils.js:307
│     throw err;
│     ^
│ Error: ENOENT: no such file or directory, stat 'C:\temp\demo\node_modules\.pnpm\simple-git-hooks@2.2.0\package.json'
│     at Object.statSync (fs.js:1086:3)
│     at _getPackageJson (C:\temp\demo\node_modules\.pnpm\simple-git-hooks@2.2.0\node_modules\simple-git-hooks\simple-git-hooks.js:140:13)
│     at checkSimpleGitHooksInDependencies (C:\temp\demo\node_modules\.pnpm\simple-git-hooks@2.2.0\node_modules\simple-git-hooks\simple-git-hooks.js:81:34)
│     at postinstall (C:\temp\demo\node_modules\.pnpm\simple-git-hooks@2.2.0\node_modules\simple-git-hooks\postinstall.js:15:9)
│     at Object.<anonymous> (C:\temp\demo\node_modules\.pnpm\simple-git-hooks@2.2.0\node_modules\simple-git-hooks\postinstall.js:24:1)
│     at Module._compile (internal/modules/cjs/loader.js:1063:30)
│     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
│     at Module.load (internal/modules/cjs/loader.js:928:32)
│     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
│     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12) {
│   errno: -4058,
│   syscall: 'stat',
│   code: 'ENOENT',
│   path: 'C:\\temp\\demo\\node_modules\\.pnpm\\simple-git-hooks@2.2.0\\package.json'
│ }
└─ Failed in 63ms
 ERROR  Command failed with exit code 1.
PS C:\temp\demo>
``` 
### Additional information:

 - `node -v` prints: v14.16.0
 - Windows, macOS, or Linux?:  tested on Windows ,and Linux

It seems that pnpm kicks off the postinstall script either too early or not in the same directory as npm would do

**Comment by erossignon** [comment]
a similar error occurs with  ```npm install git-hooks```, another package that does a post-install step.

**Comment by CxRes** [comment]
I am having similar error with my own package which has a `postinstall` step. It seems to be issue that has to do with pnpm as it works ok with npm (make sure your repo is fresh, ie no `node_modules` and `headright` is not listed as depenedency in package.json for each case while testing). 

What works:
```
npm install headright
```

What fails:
```
pnpm install headright
```

with error message:
```
node_modules/.pnpm/headright@0.1.0/node_modules/headright: Running postinstall script, failed in 35ms
.../node_modules/headright postinstall$ hr-init --ignore
│ 'hr-init' is not recognized as an internal or external command,
│ operable program or batch file.
└─ Failed in 36ms
 ERROR  Command failed with exit code 1.
```

Further local install does not even run the postinstall script:
```
pnpm i ../headright
```
exits silently!

Additional information:
node -v: v14.15.4
npm -v: tested with 6.14.11 & 7.8.0
OS: Windows 10  x64

**Comment by CxRes** [comment]
Ref #3093 who also seem to have an identical issue with local install.

**Comment by javier-garcia-meteologica** [comment]
On April 24 simple-git-hooks released version 2.4.1 and it fixed the issue for me

https://github.com/toplenboren/simple-git-hooks/pull/48

**Comment by mcmxcdev** [comment]
Works as expected with pnpm v7.1.2 and was reported as fixed on `simple-git-hooks` side. Can be closed.
